jms/src/main/scala/akka/stream/alpakka/jms/JmsSourceStage.scala

```diff
@@ -63,14 +63,7 @@ final class JmsSourceStage(settings: JmsSourceSettings) extends GraphStage[Sourc
             consumer.setMessageListener(new MessageListener {
               override def onMessage(message: Message): Unit = {
                 backpressure.acquire()
-                try {
-                  message.acknowledge()
-                  handleMessage.invoke(message)
-                } catch {
-                  case e: JMSException =>
-                    backpressure.release()
-                    handleError.invoke(e)
```

Comment:
I'm missing something here, why did the error handling go away?

Reply:
hm, I removed it because this was an exception thrown by `message.acknowledge()` which is not executed now here

Reply:
Ah, so `invoke` will never throw anything? Then it's ok.